Arrowgene.Ddon.GameServer/Handler/MailMailGetListHeadHandler.cs

```diff
@@ -1,25 +1,22 @@
-﻿using Arrowgene.Ddon.GameServer.Dump;
+using System.Linq;
 using Arrowgene.Ddon.Server;
-using Arrowgene.Ddon.Server.Network;
-using Arrowgene.Ddon.Shared.Network;
+using Arrowgene.Ddon.Shared.Entity.PacketStructure;
 using Arrowgene.Logging;
 
 namespace Arrowgene.Ddon.GameServer.Handler
 {
-    public class MailMailGetListHeadHandler : PacketHandler<GameClient>
+    public class MailMailGetListHeadHandler : GameRequestPacketHandler<C2SMailMailGetListHeadReq, S2CMailMailGetListHeadRes>
     {
         private static readonly ServerLogger Logger = LogProvider.Logger<ServerLogger>(typeof(MailMailGetListHeadHandler));
 
-
         public MailMailGetListHeadHandler(DdonGameServer server) : base(server)
         {
         }
 
-        public override PacketId Id => PacketId.C2S_MAIL_MAIL_GET_LIST_HEAD_REQ;
-
-        public override void Handle(GameClient client, IPacket packet)
+        public override S2CMailMailGetListHeadRes Handle(GameClient client, C2SMailMailGetListHeadReq request)
         {
-            client.Send(InGameDump.Dump_77);
+            // client.Send(InGameDump.Dump_77);
+            return new S2CMailMailGetListHeadRes() { Num = 1 };
```

Comment:
What does this hardcoded value do? A comment with an explanation would be useful

Reply:
This might be a mistake. I think I translated the 1 from the static packet.

Reply:
I actually have no idea what this is supposed to be. There doesn't appear to be any activity associated with this packet. Same with the footer packet. I don't see anything interesting ever happening when these packets are sent in the decrypted trace.

Reply:
Noted. A clarifying comment with this explanation in the handler would be useful for later reference when implementing the rest of the system